src/main/java/dev/hephaestus/atmosfera/client/sound/AtmosphericSoundSerializer.java

```diff
@@ -48,19 +48,14 @@ public void reload(ResourceManager manager) {
                 JsonObject json = parser.parse(new InputStreamReader(manager.getResource(resource).getInputStream())).getAsJsonObject();
 
                 Identifier soundId = new Identifier(JsonHelper.getString(json, "sound"));
-                SoundEvent sound = Registry.SOUND_EVENT.containsId(soundId)
-                        ? Registry.SOUND_EVENT.get(soundId)
-                        : Registry.register(Registry.SOUND_EVENT, soundId, new SoundEvent(soundId));
```

Comment:
Here the issue was that `Registry.register(Registry.SOUND_EVENT, ...)` would fail/crash since the registry was already frozen at this point.
There was also the inherent "issue" that sound events can never be unregistered, so loading and unloading the resource pack may have had some side effects.

What's the solution?
Turns out registering sound events is only needed for server triggered sounds which we do not use, hence we just keep the id around and create a local sound event when we wanna play the sound.